src/lib/orm/query.go

```diff
@@ -20,10 +20,15 @@ import (
 	"strings"
 
 	"github.com/astaxie/beego/orm"
-	"github.com/goharbor/harbor/src/common/dao"
 	"github.com/goharbor/harbor/src/lib/q"
 )
 
+// Params ...
+type Params = orm.Params
+
+// QuerySeter ...
+type QuerySeter = orm.QuerySeter
+
 // QuerySetter generates the query setter according to the query. "ignoredCols" is used to set the
 // columns that will not be queried
 func QuerySetter(ctx context.Context, model interface{}, query *q.Query, ignoredCols ...string) (orm.QuerySeter, error) {
```

Comment:
generall, please add more comment for the newly added method, otherwise, it's hard to read.

Reply:
+1, please add more comment to explain the mechanism of the query setter, this is helpful for consumers to use it and for the providers to know how to implement it's own filter logic

Reply:
Such as how the method should be defined for the model that is queried